### PR TITLE
[skip ci] Add missing news / merge maintenance branch into main following 2.15.7 release

### PR DESCRIPTION
**Pull request by jacobtylerwalls**
Follow-up to ea78827.

### CODE REVIEW

File: ChangeLog [hunk 1]
```diff
@@ -207,10 +207,14 @@ Release date: TBA
...
 
+* Fix false positives for ``no-member`` and ``invalid-name`` when using the ``_name_``, ``_value_`` and ``_ignore_`` sunders in Enums.
+
+  Closes pylint-dev/pylint#9015
+
 
 What's New in astroid 2.15.7?
 =============================
-Release date: TBA
+Release date: 2023-09-23
```
Comment (Pierre-Sassoulas) [inline ChangeLog:217]: I thought this should be brought by the maintenance branch merge back in main ? (https://github.com/pylint-dev/astroid/blob/main/doc/release.md#releasing-a-patch-version)

**Comment by Pierre-Sassoulas** [comment]
The merge of the maintenance branch into main is pretty hardcore (working on it).

**Review by Pierre-Sassoulas — APPROVED** [review]

**Comment by jacobtylerwalls** [comment]
> The merge of the maintenance branch into main is pretty hardcore (working on it).

Thanks, I probably made this worse by manually making edits instead of merging, but it seemed all that was needed was adding the date?

**Comment by Pierre-Sassoulas** [comment]
Yeah I realized I could have just ``git checkout . --ours`` after merging everything one by one (wrongly) then checking the diff to ``origin/main`` and fixing each previous mistake one by one 😄

**Pull request merged**